koku/api/report/test/gcp/test_gcp_query_handler.py

```diff
@@ -1274,12 +1274,16 @@ def test_query_table(self):
                 GCPCostSummaryByServiceP,
             ),
             (
-                "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,Application%20Gateway",  # noqa: E501
+                (
+                    "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRouteLoad%20Balancer"
+                    "Application%20Gateway"
+                ),
                 GCPCostView,
                 GCPCostSummaryByServiceP,
             ),
             (
-                "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,Application%20Gateway&group_by[account]=*",  # noqa: E501
+                "?filter[service]=Virtual%20Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,"
+                "Application%20Gateway&group_by[account]=*",
                 GCPCostView,
```

Comment:
This is a simpler form of doing implicit string concatenation, but it may look like a bug to a future reader. Leave a comment stating this is intentional, or wrap it in an expression as above.